airflow/providers/amazon/aws/hooks/redshift_cluster.py

```diff
@@ -157,7 +157,7 @@ def create_cluster_snapshot(
         )
         return response["Snapshot"] if response["Snapshot"] else None
 
-    def get_cluster_snapshot_status(self, snapshot_identifier: str, cluster_identifier: str):
+    def get_cluster_snapshot_status(self, snapshot_identifier: str):
```

Comment:
Good point. The reason I removed the `cluster_identifier` is because the `describe_cluster_snapshots` function throws an `InvalidParameterCombination` when both the `cluster_identifier` and `snapshot_identifier` are provided. 
```
botocore.exceptions.ClientError: An error occurred (InvalidParameterCombination) when calling the DescribeClusterSnapshots operation: Cannot specify both a Cluster identifier and a snapshot identifier.
```
I'll add the `cluster_identifier` back to the function definition, and add a deprecation warning to it.